deploy/addons/coredns/coreDNS-svc.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: coredns
+  name: kube-dns
```

Comment:
FWIW; you might wanna share the code between the kube-dns & CoreDNS Services as they should be kinda equal
not very important point though...